src/test/kotlin/com/tschuchort/compiletesting/KotlinCompilationTests.kt

```diff
@@ -645,6 +669,12 @@ class KotlinCompilationTests {
 			return fail<Nothing>("Class $className could not be loaded")
 		}
 	}
+
+	private fun classpathOf(dependency: String): File {
```

Comment:
Some documentation for this method would be nice.

Reply:
Done!

Reply:
Done!